vm/runtime.ts

```diff
@@ -421,7 +421,7 @@ module J2ME {
     ctx: Context;
     allCtxs: Set<Context>;
 
-    isolate: com.sun.cldc.isolate.Isolate;
+    isolate: any;
```

Comment:
> It is defined at https://github.com/mozilla/j2me.js/blob/master/bindings.ts.in#L428

Aha! I updated the binding in that file but missed the *com.sun.cldc.isolate.Isolate* declaration, since the file doesn't declare that type directly but rather declares a *com.sun.cldc.isolate* module with an *Isolate* interface. Fixed in d61cb1f.